### PR TITLE
SEN-1022 -- We messed up the order of merging pull requests

### PR DESCRIPTION
**Pull request by notxt**
This code has already been reviewed

### CODE REVIEW

**Comment by nackjicholson** [comment]
@noTxt @cberube  Any idea why the travis build fails? May want to try getting it to rerun.

File: dynamoDbQueueSource.js [hunk 1]
```diff
@@ -1,76 +1,106 @@
...
+  dynamoDbQueueSource._processResult = function(err, data, task) {
+    // Did we have an error? If so, pass it through to the callback, indicate
+    // that we should stop iterating, and return
+    if (err) {
+      task.error = err;
+      task.callback(err);
+      return;
+    }
+
+    // Push the items from the scan into the queue and update the total item
```
Comment (nackjicholson) [inline dynamoDbQueueSource.js:17]: The comments are slightly out of date, they only refer to the scan but this function processes results for scans and queues now.

File: dynamoDbQueueSource.js [hunk 2]
```diff
@@ -1,76 +1,106 @@
...
+  ) {
+    var task = {
+      queue: queue,
+      options: options,
+      callback: callback,
+      hasMoreItems: true,
+      results: { itemCount: 0 }
+    };
+
+    // The callback for scanToQueue is optional, but everything else is simpler
```
Comment (nackjicholson) [inline dynamoDbQueueSource.js:64]: generalize comments.

**Comment by cberube** [comment]
Closing this again -- newer, cleaner PR has been opened (#3)

**Pull request closed without merging**